bedrock/firefox/templates/firefox/mobile-en.html

```diff
@@ -0,0 +1,218 @@
+      <div class="mzp-l-content">
+        <div class="mzp-c-hero-body">
+          <h1 class="mzp-c-hero-title"><span class="c-wordmark t-firefox">Firefox Browser</span></h1>
+
+          <div class="mzp-c-hero-desc">
+            <p>{{ _('The mobile browser built for you') }}</p>
+            <p>{{ _('Take Firefox speed and privacy with you everywhere.') }}</p>
+          </div>
+
+          <div class="mzp-c-hero-cta">
+            <div class="mobile-download-buttons-wrapper">
+              <ul class="mobile-download-buttons firefox" id="mobile-download-buttons-firefox">
+                <li class="android">
+                  {{ google_play_button(href='https://app.adjust.com/2uo1qc?campaign=desktop_xp&amp;adgroup=Google_Play_Store&amp;creative=badge_en', id='playStoreLink') }}
+                </li>
+                <li class="ios">
+                  <a id="appStoreLink" href="https://app.adjust.com/2uo1qc?campaign=desktop_xp&amp;adgroup=iOS_App_Store&amp;creative=badge_en&amp;fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8" data-link-type="download" data-download-os="iOS">
+                    <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+                  </a>
+                </li>
+              </ul>
+            </div>
+            <div class="header-product-ctas">
+              <a type="link"
+                      class="mzp-c-cta-link mzp-t-product js-mobile modal-button"
+                      id="get-firefox"
+                      data-product="firefox"
+                      data-button-name="Send to SMS Modal - Firefox Mobile">{{ _('Get Firefox Mobile') }}<img class="get-firefox-icon" src={{ static('img/firefox/mobile/protocol/icon-send-device.svg') }}></a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <section class="c-private t-private mzp-t-firefox">
+      <div class="mzp-l-content">
+        <h2>{{ _('Stay invisible on the go with Private Browsing mode') }}</h2>
+        <div class="c-private-cols">
+          <div class="c-private-col">
+            <img src="{{ static('img/firefox/mobile/protocol/screen-private.svg') }}" height="331" width="163">
+            <h3>{{ _('Keep sensitive searches private') }}</h3>
+            <p>{{ _('Leave no trace with <a href="%s">Private Browsing mode</a>. When you close out, your history and cookies are deleted.')|format(url('firefox.features.private-browsing')) }}</p>
+          </div>
+          <div class="c-private-col">
+            <img src="{{ static('img/firefox/mobile/protocol/screen-tracking.svg') }}" height="331" width="163">
+            <h3>{{ _('Stop advertisers from following you') }}</h3>
+            <p>{{ _('Stay off their radar with <a href="%s">Firefox Tracking Protection</a>, automatically on in Private Browsing mode.')|format(url('firefox.features.adblocker')) }}</p>
```

Comment:
The part that says `automatically on in Private Browsing mode` is no longer true I think - it's on by default now, right?